tests/regression/roles/sonic_interfaces/defaults/main.yml

```diff
@@ -82,9 +88,11 @@ tests:
         description:
       - name: "{{ interface8 }}"
         mtu: 6000
+        unreliable_los: UNRELIABLE_LOS_MODE_OFF
       - name: "{{ interface10 }}"
         enabled:
       - name: "{{ interface9 }}"
+        unreliable_los: UNRELIABLE_LOS_MODE_ON
```

Comment:
To test this deletion, line 60 will need to set UNRELIABLE_LOS_MODE_ON for this interface.

Reply:
Thanks for making this change.

Resolved.